### ISSUE TITLE
HTTP 400 error when generating card token

### ISSUE THREAD

**Issue by BOAndrew** [hipay/hipay-fullservice-sdk-android#41]
I get "An error occurred: Your request could not be completed" every time I attempt to make a payment. It may not be SDK related because this happens in versions 1.0.1, 1.0.2 and 1.1.0 (I didn't tested v1.0.0).
However everything is working fine with iOS SDK.

The payment process fails at card token creation:
https://stage-secure-vault.hipay-tpp.com/rest/v1/token/create

The httpResult.errorStream looks like this:
```
{  
   "code":"410",
   "message":"cardNumber format expected integer but was : ",
   "description":"- code : 410 - msg : cardNumber format expected integer but was : ",
   "brand":"",
   "pan":"",
   "card_holder":"",
   "card_expiry_month":"",
   "card_expiry_year":"",
   "issuer":"",
   "country":"",
   "domesticNetwork":""
}
```

**Comment by nfillion21** [comment]
Sorry for the inconvenience, this is resolved in version 1.1.1

**Comment by nfillion21** [comment]
Sorry for the inconvenience, this is resolved in version 1.1.1
